<?xml version="1.0" encoding="utf-8"?>
<!--launch file for smoke package.-->
<launch>

  <!-- ROS and network parameter files -->
  <arg name="ros_param_file" default="$(find smoke)/config/darknet_svm.yml"/>

  <!-- Load parameters -->
  <rosparam command="load" ns="smoke" file="$(arg ros_param_file)"/>

  <include file="$(find smoke)/launch/darknet_ros_smoke.launch"></include>

  <!--video stream publisher-->
  <node pkg="smoke" type="videostr" name="video_stream_node" output="screen" required="true"/>

  <node pkg="smoke" type="smoke_nn_server.py" name="smoke_nn_server_node" output="log"/>

  <node pkg="smoke" type="smoke_svm_server" name="smoke_svm_server_node" output="screen" required="true"/>

  <!--darknet_detetor-->
  <node pkg="smoke" type="darknet_detector" name="darknet_detector_node" output="screen" required="true"/>

</launch>
